examples/task_get_network.py

```diff
@@ -0,0 +1,6 @@
+from metriq import MetriqClient
```

Comment:
Similar to the other comment regarding the network task graph, this example should not be added as it will not yield the expected result and is part of your other PR.